core/src/main/java/tc/oc/pgm/controlpoint/ControlPoint.java

```diff
@@ -414,10 +417,13 @@ private void dominate(Competitor dominantTeam, Duration dominantTime) {
       // Point is owned and must go through the neutral state before another team can capture it
       if (dominantTeam == this.controllingTeam) {
         // owner is recovering the point
-        recover(dominantTeam, dominantTime);
+        recover(dominantTeam, dominantTime, contested);
       } else if (dominantTeam != null) {
         // non-owner is uncapturing the point
-        uncapture(dominantTeam, dominantTime);
+        uncapture(dominantTeam, dominantTime, contested);
```

Comment:
```suggestion
        uncapture(dominantTeam, dominantTime);
```
We can statically deduce that contested must always be false here, if there's a dominant team there is not a contesting situation occuring. Uncapture can just call back on dominate with remaining time passing in a false